builder/core/data.py

```diff
@@ -555,14 +554,19 @@ class PKG_TOOLS(Enum):
         'yum_compiler_packages': ['gcc', 'gcc-c++'],
 
         'versions': {
-            '4.8': {},
+            '4.8': {
+                # ASan has been broken on 4.8 GCC version distributed on Ubuntu
+                # and will unlikely to get fixed upstream. so turn it off.
+                'cmake_args': ['-DENABLE_SANITIZERS=OFF'],
```

Comment:
just a thought for the future...

we might want to break some of these cmake flags out to be their own "variables", and do data overriding like `'enable_sanitizers': False`. Then in cmake.py we look for those specific variables to put in their corresponding flag. Like how we have the `run_tests` variable that turns into `-DENABLE_TESTS`.

That would be easier to manage than the current state of repeatedly appending `['-DENABLE_SANITIZERS=ON`, '-DENABLE_SANITIZERS=OFF']`, or clearing the whole list via `'cmake_args=[]` because you don't like 1 particular setting.

An alternate idea, would be a map of cmake settings, instead of a list. So like `cmake_settings: {'ENABLE_SANITIZERS': False}` and then we overlay the maps, instead of appending lists.

Reply:
Yeah, i like that idea. we likely still need to keep cmake_args for cases where precedence matters (adding headers, libs to link) and then for compile options we can have a property bag kinda of thing that gets merged as we iterate over deps